test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTest/README.md

```diff
@@ -0,0 +1,16 @@
+# OTLP exporter integration tests
+
+This directory contains a suite of integration tests that can be run using
+docker compose.
+
+Run the following command from the root of the repository to run the
+integration tests locally:
+
+```shell
+docker-compose \
```

Comment:
Could we move the filters after `docker-compose up`?

Reply:
Also, we should consider using the docker compose V2 which uses `docker compose` as the command instead of `docker-compose`.

https://github.com/docker/compose/tree/v2#about-update-and-backward-compatibility

Reply:
Changed to `docker compose`. The command will only work with the parameters in this order.